packages/account-v2/src/containers/AccountLimitsContainer/AccountLimitsTable.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { StandaloneCircleInfoRegularIcon } from '@deriv/quill-icons';
+import { Table, Text } from '@deriv-com/ui';
+import { CATEGORY, TAccountLimitValues } from '../../utils/accountLimitsUtils';
+
+const RenderRow = ({ row }: { row: TAccountLimitValues }) => {
+    const { category, hintInfo, isLessProminent, title, value } = row;
+    return (
+        <div className='grid grid-flow-col justify-between'>
+            <div>
+                {title && (
+                    <Text
+                        className={category === CATEGORY.submarket ? 'px-16' : ''}
+                        color={isLessProminent ? 'less-prominent' : 'general'}
+                        size={category === CATEGORY.footer ? 'xs' : 'sm'}
+                        weight={category === CATEGORY.header ? 'bold' : ''}
+                    >
+                        {title}
+                    </Text>
+                )}
+                {hintInfo && (
+                    <span className='px-8'>
+                        <StandaloneCircleInfoRegularIcon fill='#b5abab' iconSize='md' />
```

Comment:
Can use tailwind `fill-*` class here. If it's work then you don't need to hard code the color code.